epel8/python-specfile.spec

```diff
@@ -7,7 +7,7 @@ in a minimal diff.}
 %global base_version 0.25.1
 %global prerelease   rc1
 
-%global package_version %{base_version}%{?prerelease:~%{prerelease}}
+%global package_version 0.26.0
```

Comment:
It would be better to do it like this (I have to figure out why [prepare-release](https://github.com/packit/prepare-release) doesn't do it automatically):

```spec file
%global base_version 0.26.0
#global prerelease   rc1

%global package_version %{base_version}%{?prerelease:~%{prerelease}}
```